packages/cli/src/auth-gcp.ts

```diff
@@ -0,0 +1,58 @@
+import path from 'node:path';
+import { GoogleAuth } from 'google-auth-library';
+import {
+  fs,
+  question,
+  echo,
+} from 'zx';
+
+// https://developers.google.com/identity/protocols/oauth2/service-account?hl=pt-br
+
+const readFile = (pathFile: string): Promise<any | null> => new Promise((resolve) => {
+  try {
+    const file = fs.readJSONSync(pathFile);
+    resolve(file as any);
+  } catch (e) {
+    resolve(null);
+  }
+});
+
+export default async (projectId: string, pwd: string) => {
+  await fs.ensureDir(path.join(pwd, '.cloudcommerce'));
+  const pathAccountKeys = path.join(pwd, '.cloudcommerce', 'account-keys.json');
+  const accountKeys = await readFile(pathAccountKeys);
+
+  if (!accountKeys || !accountKeys?.clientId
+    || !accountKeys?.clientSecret || !accountKeys?.refreshToken) {
+    await echo`-- Get the Google administrator account credentials.
+    (i) To obtain them, access (https://shell.cloud.google.com/?fromcloudshell=true&show=terminal)
+
+    (ii) Execute 'gcloud auth application-default login' in cloud shell
+
+    (iii) Then access the created credentials file, to obtain them`;
+  }
+
+  const clientId = accountKeys?.clientId || await question('clientId: ');
+  const clientSecret = accountKeys?.clientSecret || await question('clientSecret: ');
+  const refreshToken = accountKeys?.refreshToken || await question('refreshToken: ');
```

Comment:
```suggestion
  const clientId = accountKeys?.clientId || await question('Google Cloud client ID: ');
  const clientSecret = accountKeys?.clientSecret || await question('Client secret: ');
```

Precisa mesmo de `refreshToken`?

Reply:
Refresh token é pra renovar o access token, vamos operar na API uma única vez, então não devemos precisar renovar...

Reply:
Ah sim, pensei que fosse possível a gnt utilizar mais de uma vez aqui 

Então nem precisa dessas credenciais todas, acredito ser possível pegar o acess token direto no gcloud usando o sheelcloud do google.

Reply:
Isso

Reply:
No deploy só usamos a service account key , só no setup que precisa de credenciais da conta pra criar a service account key...